code_challenges.rb

```diff
@@ -1,22 +1,55 @@
 # ASSESSMENT 4: Ruby Coding Practical Questions
 
 # --------------------1) Create a method that takes in an array and returns an array with only odd numbers sorted from least to greatest.
-
-
-
-
-
 fullArr1 = [4, 9, 0, '7', 8, true, 'hey', 7, 199, -9, false, 'hola']
 # Expected output: [-9, 7, 9, 199]
 fullArr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
 
+#  I cant get the number 199 to show up when printing for odds, ive tried researching and nothing.
+def odd_values(arr)
+    arr.select.with_index { |_, i| i.odd? }
+end
 
-# --------------------2) Create a class called Bike that is initialized with a model, wheels, and a frame size. The default number of wheels is 2. Create a get_info method that returns a sentence with all the data from the bike object.
-
-
+p odd_values(fullArr1)
+p odd_values(fullArr2)
```

Comment:
so right now this is not returning the expected result. for `fullArr1` we get `[9, "7", true, 7, -9, "hola"]`. this code would need to be refactored.